### DISCUSSION TITLE
Interpretation of species vectors in nMDS

### DISCUSSION THREAD

**Discussion by friendly** [vegandevs/vegan#639] [Q&A]
I posed this question on SO, [Interpretation of species vectors in nMDS](https://stackoverflow.com/questions/78342759/interpretation-of-species-vectors-in-nmds), but have had no answers, so I'm raising it here.

I'm trying to compare results I get from a PCA biplot with what I can get from `MASS:isoMDS()` and `vegan::metaMDS()` particularly using vectors representing the variables to help interpret the MDS dimensions.

My question is essentially how I can understand what is returned by the `$species` component of `metaMDS()`. How is it calculated and what do the values represent? 

The plots I make for PCA and `isoMDS()` are consistent with the interpretation that the variable vectors reflect correlations of the observed variables with the dimensions. The plot for `metaMDS()` is inconsistent for on of the variables (`relwt`) which points in the opposite direction.

Here is the plot from `ggbiplot::ggbiplot()`

https://i.stack.imgur.com/4jD0N.png

Here is what I get from `metaMDS()`

https://i.stack.imgur.com/8n99m.png

**Comment by jarioksa** [comment]
This is what `help(metaMDS)` says:
```
       6. Species scores: Function adds the species scores to the final
          solution as weighted averages using function ‘wascores’ with
          given value of parameter ‘expand’.
```
So they are not vectors but weighted averages.

If you want to get vectors, try `envfit` / `vectorfit`.
